src/helm-frontend/src/services/getSchema.ts

```diff
@@ -6,10 +6,12 @@ import getBenchmarkRelease from "@/utils/getBenchmarkRelease";
 
 export default async function getSchema(signal: AbortSignal): Promise<Schema> {
   try {
+    // TODO: this should not be hard-coded to the main website, want to be able
+    // to still run things locally.
     const resp = await fetch(
       `https://crfm.stanford.edu/helm/${
         getBenchmarkRelease() || getBenchmarkSuite()
-      }/schema.yaml`,
+      }/schema.json`,
```

Comment:
This causes `parse(data)` to break later on. Just deserialize as JSON instead of parsing as YAML.

Also, why the change from YAML to JSON? I agree that we need to do it eventually, but that seems not relevant to the rest of this PR. This will cause some pain for developers switching between branches because this frontend won't work with the `helm-summarize` output from the other branches.  I have some idea for how to add a shim for this, which I can add in a follow-up PR.

Reply:
Oops, fixed; thanks for the catch.  We need to break backward compatibility anyway, since the schema file is now in the suite directory, so I figured this is a good time to just switch over to JSON.